<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>34</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>128</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>8</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QLabel" name="idxLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="description">
     <property name="minimumSize">
      <size>
       <width>128</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="name"/>
   </item>
   <item>
    <widget class="QComboBox" name="dataType">
     <property name="minimumSize">
      <size>
       <width>128</width>
       <height>0</height>
      </size>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>INTEGER</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NUMBER(16,2)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CHAR(20)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>VARCHAR(200)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TEXT</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
